tv_apps/browser/style/index.css

```diff
@@ -1,15 +1,21 @@
-
 html, body {
   width: 1920px;
   height: 100%;
   padding: 0;
   margin: 0;
-  font-size: 20px;
   overflow: hidden;
   cursor: default;
   background-color : #e0e2e4;
 }
 
+html {
+  font-size: 10px;
+}
+
+body {
+  font-size: 20px;
+}
+
```

Comment:
To reset the default rem to 10px. Our components will be affected if the default rem is 20px. This also better aligns to our overall CSS settings. Leave body to 20px because our partner uses it extensively in the app.